### ISSUE TITLE
[Feature]: Support for/Integration with Forge UI

### ISSUE THREAD

**Issue by LichAcademy** [continue-revolution/sd-webui-segment-anything#202]
### Expected behavior

Hi. Sorry, not a programmer, apologies if I am using this GitHub platform incorrectly.

I couldn't get this extension to work until I realised it's not the plugin, but the UI. Text-2-image works fine on A1111, but not so much on Forge.

Actually, as I'm typing this, I realised the issue is actually in communication between extensions. On Forge, when you tick the box `Send to ControlNet` (on text2img tab), parameters don't get passed. On A1111, they do.

Just thought I'd bring to your attention, that's all. Thank you for the extension! :)

**Comment by Rinual** [comment]
yeah, not even sure you can specify the location of the ultralytics / segm models in forge, but i could be wrong. I assume it still wont work for you though?
